src/components/table/JournalEntryTable.tsx

```diff
@@ -91,8 +92,8 @@ const JournalEntryTable = withStyles(styles)(
           const details = [];
           if (row.entryType === JournalEntryModelEntryTypeEnum.Create) {
             note = description;
-          } else if (parsedReq[0] === 'bio.terra.model.IngestRequestModel') {
-            const strategy = parsedReq[1].updateStrategy;
+          } else if (_.get(parsedReq, 0) === 'bio.terra.model.IngestRequestModel') {
```

Comment:
Can you use optional chaining here?
```suggestion
          } else if (parsedReq?.[0] === 'bio.terra.model.IngestRequestModel') {
```